client/src/components/LoginPage/index.js

```diff
@@ -9,8 +10,26 @@ const styles = makeStyles({
     margin: 10,
     width: "80%",
   },
+  button: {
+    background: 'linear-gradient(45deg, #2196F3 30%, #21CBF3 90%)',
```

Comment:
Consider using global constants for colors so that if we want to change the whole color scheme we only need to modify the theme.  CHeck out material ui ThemeProvider

Reply:
fixed it!